bundles/framework/publisher2/resources/scss/style.scss

```diff
@@ -37,11 +37,34 @@ div.notLoggedIn {
     padding: 10px;
 }
 
+div.publisher-announcements-inputs {
+    display: grid;
+    grid-column-gap: 50px;
+    grid-template-columns: auto auto;
+
+    .icon-warning-light {
+        float: right;
+        margin-left: 12px;
+    }
+}
+
```

Comment:
Move these scss-definitions to admin-announcements where the announcement tool is defined.